src/components/UpcomingEvents.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Link } from 'gatsby'
```

Comment:
Since all the links in this component will be external, no need to import `Link`